src/domUpdates.js

```diff
@@ -1,16 +1,71 @@
-//NOTE: Your DOM manipulation will occur in this file
+// Imports
+import {recipeData} from './data/recipes'
+import {recipeGrid} from './scripts'
 
-//Here are 2 example functions just to demonstrate one way you can export/import between the two js files. You'll want to delete these once you get your own code going.
-function exampleFunction1(person) {
-  console.log(`oh hi there ${person}`)
+// functions
+
+const makeRecipeColumnData = (data) => {
+  const mappedRecipe = data.map((recipe, index) => {
+    return {
+      column: index+1,
+      id: recipe.id,
+      image: recipe.image,
+      name: recipe.name,
+      tags: recipe.tags
+    }
+  });
+
+  const leftColumn = mappedRecipe.filter(recipe => recipe.column % 3 === 1);
+  const centreColumn = mappedRecipe.filter(recipe => recipe.column % 3 === 2);
+  const rightColumn = mappedRecipe.filter(recipe => recipe.column % 3 === 0);
+  return [leftColumn, centreColumn, rightColumn];
 }
 
-function exampleFunction2(person) {
-  console.log(`bye now ${person}`)
+const createSingleRecipeHTML = singleRecipe => {
+  let htmlCode = '';
+  htmlCode += 
+  `
+  <div class="individual-recipe">
+      <div class="recipe-image">
+        <img src="${singleRecipe.image}">
+        <div class="hover-card"> 
+          <h3>Read more...</h3>
+        </div>               
+      </div>
+    <h2>${singleRecipe.name}</h2>
+  </div>
+  `;
+  return htmlCode
 }
 
+const createColumnHTML = column => {
+  let htmlCode = '';
+  htmlCode += '<div class = col>'
```

Comment:
I believe we are missing the quotes around 'col' here

Reply:
done